lib/index.js

```diff
@@ -13,7 +14,8 @@ const internals = {};
 
 module.exports = {
     configs: {
-        recommended: Recommended
+        recommended: Recommended,
+        internal: Internal
```

Comment:
Is `internal` named this way because it should not be used outside the organization?  Or will we consider this safe for others to consume, e.g. in the pal boilerplate? https://github.com/hapipal/boilerplate/blob/pal/.eslintrc.json

In my opinion it would be useful to expose something that others could use who want to work with lab's default config but further customize it.

Reply:
I named it internal from its current usage, since it was only used within the org. I think it can be used by everybody, there are no problem with that I think. Did you have a better name in mind? `ladDefault`? `org`? `organization`?

Reply:
What about just: `lab`?

Reply:
I initially didn't want to tie the config name to a tool since it will no longer be used only within `lab` but also throughout the organization. What about `lib` or `library`? Or perhaps it was your intention to convey that it's not a config for a special use case as "developing a library" but rather one that can be used in multiple places then it should be named as a usage?

Reply:
Of the presented names I prefer `@hapi/internal`, but I propose `@hapi/module` as a better name?

Reply:
I like `@hapi/internal` too but I'm ok with `@hapi/module` as well.